<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PI-ICR analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>PI-ICR-fitting</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>236</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Position fit cut:</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="x_max">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="y_max">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="x_min">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="y_min">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X [min, max] / ch.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y [min, max] / ch.</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>146</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>ToF cut:</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="tof_max">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tof_min">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ToF min / μs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ToF max / μs</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>146</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Z-class analysis:</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="z_class_max">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="z_class_min">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min # / shot</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max # / shot</string>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>241</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Batch</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>543</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>PI-ICR spot reconstruction and frequency calculation V2.1:

Did you run &quot;piicr_timestamps.exe&quot; to add the timestamps to the raw data?

Just put all the files (center file, p1p2 files, frequency config file [.out]) into one folder and
select it. Then provide information about the isotopes (e.g. '85Rb') and klick 'Run'.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope4">
        <property name="geometry">
         <rect>
          <x>358</x>
          <y>150</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope5">
        <property name="geometry">
         <rect>
          <x>427</x>
          <y>150</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="folder_path">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>115</y>
          <width>471</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="load">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>111</y>
          <width>72</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope1">
        <property name="geometry">
         <rect>
          <x>151</x>
          <y>150</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="run">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>146</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope3">
        <property name="geometry">
         <rect>
          <x>289</x>
          <y>150</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>150</y>
          <width>122</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter isotopes here:</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>541</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Isomer analysis: simultaneous 2D gaussian fit for both ground and isomer (P2 spot)</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_ellipse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>541</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Manual spot selection (Click+hold ellipse selector)</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Single</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>-10</y>
          <width>543</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>PI-ICR spot reconstruction and frequency calculation V2.1:

Did you run &quot;piicr_timestamps.exe&quot; to add the timestamps to the raw data?
Select a file to fit and klick 'Run'.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="load_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>80</y>
          <width>72</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QPushButton" name="run_3">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>155</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="file_names">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>471</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="isotope1_2">
        <property name="geometry">
         <rect>
          <x>495</x>
          <y>130</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>110</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Isotope:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>556</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Fix FWHM fit parameter</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>25</y>
         <width>491</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Load &quot;_all_peak_positions_c.csv&quot; of reference ion used for FWHM parameter fix</string>
       </property>
      </widget>
      <widget class="QPushButton" name="load_3">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>40</y>
         <width>72</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="folder_path_2">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>44</y>
         <width>471</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Cross-checks</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>191</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Single</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>543</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Enter REF here:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ref"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Enter MEAS here:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="meas"/>
         </item>
         <item>
          <widget class="QPushButton" name="run_2">
           <property name="text">
            <string>Go!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>526</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>PI-ICR cross-check calculation V2.1:

Just select which isotope should be the measurement ion and which one should be the
reference ion. Finally click 'Go!'.</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>105</y>
          <width>341</width>
          <height>76</height>
         </rect>
        </property>
        <property name="title">
         <string>Isomer-analysis</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <widget class="QLineEdit" name="ground_or_isomer">
         <property name="geometry">
          <rect>
           <x>165</x>
           <y>25</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>25</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Enter 'ground' or 'isomer':</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>50</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Mass excess / keV :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="isomer_mass_excess">
         <property name="geometry">
          <rect>
           <x>165</x>
           <y>50</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>50</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="isomer_mass_excess_unc">
         <property name="geometry">
          <rect>
           <x>265</x>
           <y>50</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Combined</string>
       </attribute>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>531</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>PI-ICR cross-check combination V2.1:

If you want to combine multiple cross-checks of a run or even multiple runs just select
the run-folders (same as in fitting) of the runs you want to combine and klick 'Go!'.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="run_8">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>140</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
       <widget class="QPushButton" name="load_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>110</y>
          <width>72</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="folder_paths">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>471</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit_bttn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>41</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionfdsa"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="action2017_Jonas_Karthein_Questions_comments_to_jonas_karthein_cern_ch"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfdsa">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action2017_Jonas_Karthein_Questions_comments_to_jonas_karthein_cern_ch">
   <property name="text">
    <string>2017 Jonas Karthein. Questions/comments to jonas.karthein@cern.ch</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>folder_path</tabstop>
  <tabstop>load</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>isotope1</tabstop>
  <tabstop>isotope2</tabstop>
  <tabstop>isotope3</tabstop>
  <tabstop>isotope4</tabstop>
  <tabstop>isotope5</tabstop>
  <tabstop>run</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>x_min</tabstop>
  <tabstop>x_max</tabstop>
  <tabstop>y_min</tabstop>
  <tabstop>y_max</tabstop>
  <tabstop>groupBox_2</tabstop>
  <tabstop>tof_min</tabstop>
  <tabstop>tof_max</tabstop>
  <tabstop>groupBox_3</tabstop>
  <tabstop>z_class_min</tabstop>
  <tabstop>z_class_max</tabstop>
  <tabstop>groupBox_4</tabstop>
  <tabstop>folder_path_2</tabstop>
  <tabstop>load_3</tabstop>
  <tabstop>exit_bttn</tabstop>
  <tabstop>ref</tabstop>
  <tabstop>meas</tabstop>
  <tabstop>run_2</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>load_2</tabstop>
  <tabstop>run_3</tabstop>
  <tabstop>file_names</tabstop>
  <tabstop>isotope1_2</tabstop>
  <tabstop>tabWidget_3</tabstop>
  <tabstop>run_8</tabstop>
  <tabstop>load_5</tabstop>
  <tabstop>folder_paths</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
